returnn/tf/layers/signal_processing.py

```diff
@@ -561,48 +613,51 @@ def __init__(self, noise_power_spectrum_tensor):
       def from_layer(cls, layer):
         return cls(layer.output.get_placeholder_as_batch_major())
 
-      def getNoisePowerSpectrum(self):
+      def get_noise_power_spectrum(self):
         return self._noise_power_spectrum_tensor
 
-    def _getParametersFromConstructorInputs(parameters, l_overwrite, p_overwrite, q_overwrite, average_parameters):
+    def _get_parameters_from_constructor_inputs(parameters_, l_overwrite_, p_overwrite_, q_overwrite_,
```

Comment:
In the case that you need to rename arguments of the local function, I would rather prefer to not do that. So the `# noinspection ...` is a cleaner here, I think.

Reply:
That goes for all functions, I guess?

Reply:
What do you mean?

Reply:
Then I'll revert all changes that include renaming function arguments and use `# noinspection...` instead.

But renaming is ok, so I leave that?

Reply:
I don't understand what you ask about renaming the function. The code should pass the code style check. You can name the function whatever way you like as long as it passed the code style check.

Reply:
You did not want the arguments to be renamed. Therefore, I was wondering if you want the function to be renamed. From your last comment I assume that is alright. 

I now undid the renaming of arguments and used `# noinspection` instead, but kept the updated function name.